observation/gateway.pb.go

```diff
@@ -3,8 +3,8 @@
 
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.34.2
-// 	protoc        v5.28.3
+// 	protoc-gen-go v1.35.1
```

Comment:
Not a blocker, but would be nice to agree on a specific version for both `protoc` and `proto-gen-go` to minimize the amount of unnecessary changes that show up on a PR.

Reply:
> Not a blocker, but would be nice to agree on a specific version for both `protoc` and `proto-gen-go` to minimize the amount of unnecessary changes that show up on a PR.

100% agree; I have been thinking the same thing. I looked into it a little bit and seems there are tools available for managing protobuf installations. The mostup to date and widely used seems to be Buf: https://github.com/bufbuild/buf

Worth looking into; WDYT @adshmh @Olshansk ?

Reply:
I'd ask to be consistent with `poktroll` which has other downstream dependencies (e.g. cosmos SDK).

Once we upgrade to cosmos 0.52, we can revisit this then.